pkgs/development/libraries/libnfs/default.nix

```diff
@@ -13,6 +13,18 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ autoreconfHook ];
 
+  patches = [
+    # Fixes 100% CPU usage in multi-threaded mode
+    (fetchpatch {
+      url = "https://github.com/sahlberg/libnfs/commit/34d6fe37e986da5b0ced86cd028a88e482537d5a.patch";
+      sha256 = "sha256-i7mi+TVdkLb4MztT5Ic/Q8XBIWk9lo8v5bNjHOr6LaI=";
+    })
+  ];
+
+  configureFlags = [
+    "--enable-pthread"
```

Comment:
```suggestion
    "--disable-werror"
    "--enable-pthread"
```

Reply:
@SuperSandro2000 I've made a PR to `libnfs` to fix the deprecation warnings on modern macOS: https://github.com/sahlberg/libnfs/pull/403

I'll update this PR to disable `werror` on macOS if the PR is not accepted upstream.